packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/setUp.st

```diff
@@ -0,0 +1,11 @@
+as yet unclassified
+setUp
+
+	| node result |
+	self halt.
```

Comment:
Is still in here for debugging reasons?

Reply:
At the moment for unexplainable reason the tests are just working if we stop the execution and then continue it. We will discuss this with Eva and Patrick.